Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14.17-alpine3.14
+
+LABEL application=airkeeper \
+    description="Airkeeper lambda function"
+
+RUN apk add --update --no-cache git python3 make g++\
+    && rm -rf /var/cache/apk/*
+
+RUN git clone --single-branch --branch main https://github.com/api3dao/airkeeper.git
+
+WORKDIR /airkeeper
+
+RUN npm install
+
+CMD npm run sls:config \
+    && npm run sls:$COMMAND
```

Comment:
I see multiple issues:
* You're running the command as root. That's usually a bad practice for Docker containers. You should [create a new user and use it to run the commands](https://github.com/api3dao/airnode/blob/07d540f61d3fe29a0213d8cf80046856727b4874/packages/airnode-admin/docker/Dockerfile#L19).
* By running `npm install` you're installing all the development dependencies and the resulting image is huge, more than 700MB
* Using an env variable to provide a command is quite an unusual approach. Usually with the CLI containers (which this essentially is) you use it like any other CLI tool, in this case, it would be for example `docker run ... api3/airkeeper deploy`.

I would suggest installing `serverless` as a global package in the container so that you don't need to install dev dependencies and writing a small `entrypoint.sh` that will pass the correct command from the command line instead of the env variable.

Reply:
All valid points and I'm going to do some research to start applying those suggestions and I guess I used the env variable for the command because the idea was to come up with something similar to pre-alpha deployer [Dockerfile](https://github.com/api3dao/airnode/blob/pre-alpha/Dockerfile).

Reply:
These will be resolved by https://api3dao.atlassian.net/browse/BEC-95